test/library/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+set -x
+
+git clone https://github.com/weaveworks-gitops-test/wego-library-test.git test/library/wego-library-test
+cd test/library/wego-library-test
+npm ci
```

Comment:
When running this locally, it fails with 
> npm ci
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@weaveworks/weave-gitops/0.0.4/fc8d34755955e02e2004edab9e4dd88f22aaf46b933498f048f0abb256f87644

Is it intentional that it pulls package version 0.0.4?  This surprised me (but I don't completely grasp how the test verifies the change in APIs).  Please add some comments in the Makefile describing what this is checking.  It might be worthwhile to add a bit of info to the readme as well.

Reply:
> Is it intentional that it pulls package version 0.0.4? 
I hadn't realized that, but this test also doesn't seek to prove the UI stuff works (we have some  test coverage on that already). That begs the question as to why we are installing the JS module in the test repo.

This comment gave me an idea of how to include the JS stuff in this integration test. I'll make a time-boxed effort to get that working.

Reply:
Gave up on adding `index.ts` coverage here, it wasn't working, and we have unit test coverage there.

Reply:
The problem with the GH token is it requires read:packages permission